ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/ShowOrgBasemaps/ShowOrgBasemaps.h

```diff
@@ -64,10 +63,11 @@ class ShowOrgBasemaps : public QQuickItem
 
 private:
   void load();
+  void connectLoadStatusSignal();
 
   Esri::ArcGISRuntime::Map* m_map = nullptr;
   Esri::ArcGISRuntime::MapQuickView* m_mapView = nullptr;
-  Esri::ArcGISRuntime::Portal* m_portal = nullptr;
+  std::shared_ptr<Esri::ArcGISRuntime::Portal> m_portal = nullptr;
```

Comment:
This seems unrelated to solving the problem at hand. Typical usage is raw pointers and specifying `this` as the parent.

Reply:
Used raw pointer.